.github/workflows/build-wheels.yml

```diff
@@ -202,24 +217,19 @@ jobs:
         echo ::group::Set up dependencies
         python --version
         python -c "import struct; print(struct.calcsize('P') * 8)"
-
         python -m pip install -U pip setuptools wheel wheelhouse_uploader ${{ env.BUILD_DEPENDS }}
         echo ::endgroup::
-
```

Comment:
Any reason to remove the existing blank lines?

Here and in many other places in this PR.

Reply:
i'm sorry

forgot to check the formatting
change it now